draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -151,7 +151,11 @@ This is especially useful in certain conferencing topologies, where otherwise se
 
 ### Multiplexing RTP, RTCP, and Non-RTP Flows on a Single QUIC Connection {#single-path}
 
-In order to conserve ports, especially at NATs and Firewalls, this specification defines a flow identifier, so that multiple RTP flows, RTCP flows, and non-RTP flows can be distinguished even if they are carried on the same QUIC connection. This is described in more detail in {{multiplexing}}.
+This specification defines a flow identifier for multiplexing multiple RTP and
+RTCP ports on the same QUIC connection to conserve ports, especially at NATs and
+Firewalls. {{multiplexing}} describes the multiplexing in more detail. Future
+extensions could further build on the flow identifier to multiplex RTP/RTCP with
+other protocols on the same connection.
```

Comment:
> Future extensions could further build on the flow identifier to multiplex RTP/RTCP with other protocols on the same connection.

I think it's worth pointing out (as we have realized while working on this specification) that there's more to multiplexing than just disambiguating and demultiplexing. Perhaps 

> Future extensions could further build on the flow identifier to multiplex RTP/RTCP with other protocols on the same connection, **as long as these protocols can co-exist with RTP/RTCP without interfering with the ability of this connection to carry real-time media.** 

Does that make sense?

Reply:
Makes sense. I updated the PR.